corehq/ex-submodules/couchexport/tests/test_saved.py

```diff
@@ -49,6 +49,18 @@ def test_get_saved_and_delete_copies_multiple(self):
         self.assertEqual(1, len(saved_after_deletion))
         self.assertEqual(chosen_one._id, saved_after_deletion[0]._id)
 
+    def test_save_basic_export_to_blobdb(self):
+        index = ['single']
+        saved_export = SavedBasicExport(configuration=_mk_config(index=index))
+        saved_export.save()
+        saved_export.set_payload("content")
+        name = saved_export.get_attachment_name()
+        assert saved_export.has_file()
```

Comment:
again mixing `assert` with unit test `self.assertX`. I realize that this may not be a test assert and just an assumption assert in which case probably fine.